crates/sui-framework/packages/sui-framework/sources/package.move

```diff
@@ -9,6 +9,51 @@ module sui::package {
     use std::type_name;
     use sui::types;
 
+    /// Allows calling `.burn` to destroy a `Publisher`.
+    public use fun burn_publisher as Publisher.burn;
+
+    /// Allows calling `.module_` to access the name of the module a
+    /// `Publisher` was derived from.
+    public use fun published_module as Publisher.module_;
```

Comment:
```suggestion
    public use fun published_module as Publisher.`module`;
```
Thoughts? Feelings?

Reply:
I have ...mixed feelings, but not very strong, so happy to be persuaded.

Reply:
Will land, but let me know if you do feel strongly about using the backticks for this (for me it makes the receiver syntax fiddlier/more conspicious than the non-receiver function, vs tacking an underscore on the end).